docs/source/changes.rst

```diff
@@ -1,17 +1,50 @@
 Changelog
 =========
 
-v2.4 (Development)
+v2.5 (Development)
```

Comment:
I typically wait to add this section in in a following commit so the readthedocs "stable" docs doesn't have it

Reply:
good call, thanks